### DISCUSSION TITLE
Autocompleting like WebStorm

### DISCUSSION THREAD

**Discussion by fbpzrl** [microsoft/vscode-discussions#539] [Extension Show and Tell]
Hello!
I'm looking for a VS Code extension that suggests autocompleting class name from HTML document or IDs when I'm programming with JavaScript, just like WebStorm does.

WebStorm can "know" the project and suggest the properties contained in the HTML, for example:
In main.js
`document.querySelector('.product')`
WebStorm can suggest the "product class" from HTML.

**Comment by int-arsh** [comment]
I am also looking for the same If you already got it, could you let me know?
